docs/js/alert.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Alert
+layout: page.jade
+sidebar: true
+collection: js
+priority: 3
+path: alert
+---
+
+# Alert
+<p class="lead">The Alert component is an empty container that you input a message and have an OK button by default.</p>
+
+An alert box is often used if you want to make sure information comes through to the user.
+
+### How to use
+
+You can initiate the component as a jQuery plugin:
+
+```js
+// using a data attribute
+$('[data-alert]').on('click', () => $.fn.alert('this is an example message'))
+
+// using a class
+$('.data-alert').on('click', () => $.fn.alert('this is an example message'))
+```
+or a vanilla constructor:
+
+```js
+import Alert from 'garden/src/js/components/alert';
+
+new Alert('message');
+```
+
+Working Example:
+
+<button class="button button-primary" data-alert>Open Alert</button>
+
+### Notes
+
+The alert dialog should be used for messages which do not require any response on the part of the user, other than the acknowledgement of the message.
```

Comment:
Aqui eu usaria `that` ao invés de `which`. Existe uma pequena diferença: which geralmente é usado em orações explicativas onde tu tá apenas descrevendo um nome, já o `that` é usado pra orações restritivas, onde tu limita aquele nome a um certo grupo de características, que é o caso aqui.